Java/core/src/main/java/com/amazon/randomcutforest/RandomCutForest.java

```diff
@@ -290,8 +290,7 @@ public RandomCutForest(Builder<?> builder) {
 
     private void initCompactDouble(Builder<?> builder) {
         PointStoreDouble tempStore = PointStoreDouble.builder().internalRotationEnabled(builder.internalRotationEnabled)
-                .capacity(numberOfTrees * sampleSize + 1).indexCapacity(builder.initialPointStoreSize.get())
-                .currentStoreCapacity(builder.initialPointStoreSize.get())
+                .capacity(numberOfTrees * sampleSize + 1).initialSize(builder.initialPointStoreSize.get())
```

Comment:
How does `initialPointStoreSize` get initialized if the user doesn't supply a value?

Reply:
It's set to 2*sampleSize earlier. But can be solved cleaner - changing.